Cargo.toml

```diff
@@ -18,31 +18,31 @@ license = "MIT"
 license-file = "LICENSE"
 
 [workspace.dependencies]
-blockifier = { git = "https://github.com/starkware-libs/blockifier.git", rev = "bd43cc3" }
-starknet_api = "0.5.0-rc1"
-cairo-lang-casm = "=2.3.1"
-cairo-lang-runner = "=2.3.1"
-cairo-lang-sierra-to-casm = "=2.3.1"
-cairo-lang-sierra = "=2.3.1"
-cairo-lang-utils = "=2.3.1"
-cairo-lang-starknet = "=2.3.1"
-cairo-lang-compiler = "=2.3.1"
-cairo-lang-filesystem = "=2.3.1"
-cairo-lang-debug = "=2.3.1"
-cairo-lang-defs = "=2.3.1"
-cairo-lang-diagnostics = "=2.3.1"
-cairo-lang-lowering = "=2.3.1"
-cairo-lang-plugins = "=2.3.1"
-cairo-lang-project = "=2.3.1"
-cairo-lang-semantic = "=2.3.1"
-cairo-lang-sierra-ap-change = "=2.3.1"
-cairo-lang-sierra-gas = "=2.3.1"
-cairo-lang-sierra-type-size = "2.3.1"
-cairo-lang-sierra-generator = "=2.3.1"
-cairo-lang-syntax = "=2.3.1"
-cairo-lang-test-plugin = "=2.3.1"
+blockifier = { git = "https://github.com/starkware-libs/blockifier.git", rev = "51b343f" }
+starknet_api = "0.6.0-rc2"
+cairo-lang-casm = "=2.4.0"
+cairo-lang-runner = "=2.4.0"
+cairo-lang-sierra-to-casm = "=2.4.0"
+cairo-lang-sierra = "=2.4.0"
+cairo-lang-utils = "=2.4.0"
+cairo-lang-starknet = "=2.4.0"
+cairo-lang-compiler = "=2.4.0"
+cairo-lang-filesystem = "=2.4.0"
+cairo-lang-debug = "=2.4.0"
+cairo-lang-defs = "=2.4.0"
+cairo-lang-diagnostics = "=2.4.0"
+cairo-lang-lowering = "=2.4.0"
+cairo-lang-plugins = "=2.4.0"
+cairo-lang-project = "=2.4.0"
+cairo-lang-semantic = "=2.4.0"
+cairo-lang-sierra-ap-change = "=2.4.0"
+cairo-lang-sierra-gas = "=2.4.0"
+cairo-lang-sierra-type-size = "2.4.0"
+cairo-lang-sierra-generator = "=2.4.0"
+cairo-lang-syntax = "=2.4.0"
+cairo-lang-test-plugin = "=2.4.0"
 cairo-vm = "0.8.2"
-cairo-felt = "0.8.2"
+cairo-felt = "=0.8.2"
```

Comment:
Why was `=` added here?

Reply:
I thought it was problem with peer dependencies, but I update it and it seems to work well